p2p/ipld/read_test.go

```diff
@@ -227,70 +120,55 @@ func TestRetrieveBlockData(t *testing.T) {
 		expectErr  bool
 		errStr     string
 	}
-
-	// issue a new API object
-	ipfsAPI := mockedIpfsAPI(t)
-
-	// the max size of messages that won't get split
-	adjustedMsgSize := consts.MsgShareSize - 2
-
 	tests := []test{
 		{"Empty block", 1, false, ""},
 		{"4 KB block", 4, false, ""},
 		{"16 KB block", 8, false, ""},
-		{"16 KB block timeout expected", 8, true, "timeout"},
+		{"16 KB block timeout expected", 8, true, "not found"},
 		{"max square size", consts.MaxSquareSize, false, ""},
 	}
 
 	for _, tc := range tests {
 		tc := tc
-
 		t.Run(fmt.Sprintf("%s size %d", tc.name, tc.squareSize), func(t *testing.T) {
-			// if we're using the race detector, skip some large tests due to time and
-			// concurrency constraints
-			if racedetector.IsActive() && tc.squareSize > 8 {
-				t.Skip("Not running large test due to time and concurrency constraints while race detector is active.")
-			}
```

Comment:
Can we delete the racedetector package then, or is it still used somewhere else?

Reply:
I thought I deleted it, but it got lost in stash

Reply:
Deleting it